README.md

```diff
@@ -2,8 +2,8 @@
 
 > The repository for *high quality* TypeScript type definitions.
 
-*You can also read this README in [Spanish](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.es.md), [Korean](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ko.md), [Russian](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ru.md), [Chinese](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.zh.md), [Portuguese](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.pt.md), [Italian](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.it.md)
-and [Japanese](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ja.md)!*
+*You can also read this README in [español](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.es.md), [한국어](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ko.md), [Русский язык](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ru.md), [汉语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.zh.md), [português](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.pt.md), [italiano](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.it.md)
```

Comment:
I'm not a native speaker of any of these languages, but I would have guessed they should be capitalized?

Reply:
Wikipedia just has "Русский" for Russian - no idea which is preferred.

Reply:
I'm having trouble confirming the text for zh because I'm not familiar with the various scripts.

Reply:
Curiously, each of the translated readmes appears to begin with a list of other languages in the language of the readme (e.g. these languages, but named in Russian).

Reply:
FWIW, I like the idea.

Reply:
I searched up online for the translations, I happen to know Chinese, so the Chinese one is definitely correct. The Italian one I am pretty confident is correct since this [Italian](http://theitalianinterpreter.co.uk/2013/04/02/italian-grammar-when-to-use-uppercase/#:~:text=As%20a%20rule%2C%20proper%20names,the%20beginning%20of%20a%20sentence.&text=In%20headings%2Ftitles%20normally%20only,title%20is%20in%20lower%20case.) grammar website says so. I'm not sure about Russian unfortunately. I am assuming that the Japanese translation is correct, since Chinese and Japanese share many of the same characters, and it looks right. I saw on a form that you lowercase the [Portuguese](https://forum.wordreference.com/threads/capital-letter-for-the-first-character-of-a-language-name.2795216/) one. I'm not sure about the Korean one, but I'm assuming it's correct. I used [this](https://omniglot.com/language/names.htm) website to get most of the translations. The Spanish one I believe is correct, because I found [this](https://www.spanish.academy/blog/types-of-spanish-words-you-shouldnt-capitalize/) site talking about it.

Reply:
> Curiously, each of the translated readmes appears to begin with a list of other languages in the language of the readme (e.g. these languages, but named in Russian).

It probably is not as big of a deal, because it is not the main README, but they probably should also be changed.

Reply:
Either way, it would be best to have someone fluent in those languages verify that it is correct.

Reply:
I have to say, I don't know anything else, but 'zh' is Chinese, and it should be ‘中文’ or '简体中文'